packages/jest-changed-files/src/__tests__/git-test.js

```diff
@@ -54,7 +54,7 @@ describe('gitChecker', () => {
     });
 
     it('returns an empty array for git repo folder without modified files', () => {
-      return git.findChangedFiles(tmpdir).then(res => {
+      return git.findChangedFiles(tmpdir, {lastCommit:false}).then(res => {
```

Comment:
please stick to our code style. Here it would be `{lastCommit: false}` and it is similar to everywhere else :)